items:
- uid: Microsoft.WindowsAzure.Storage.DataMovement.TransferException
  commentId: T:Microsoft.WindowsAzure.Storage.DataMovement.TransferException
  id: TransferException
  parent: Microsoft.WindowsAzure.Storage.DataMovement
  children:
  - Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor
  - Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode)
  - Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode,System.String)
  - Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode,System.String,Exception)
  - Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(System.String)
  - Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(System.String,Exception)
  - Microsoft.WindowsAzure.Storage.DataMovement.TransferException.ErrorCode
  langs:
  - csharp
  - vb
  name: TransferException
  fullName: Microsoft.WindowsAzure.Storage.DataMovement.TransferException
  type: Class
  source:
    remote:
      path: lib/Exceptions/TransferException.cs
      branch: master
      repo: https://github.com/Azure/azure-storage-net-data-movement
    id: TransferException
    path: lib/Exceptions/TransferException.cs
    startLine: 17
  assemblies:
  - Microsoft.WindowsAzure.Storage.DataMovement
  packages:
  - Microsoft.WindowsAzure.Storage.DataMovement(0.3.0.0)
  namespace: Microsoft.WindowsAzure.Storage.DataMovement
  summary: "\nBase exception class for exceptions thrown by Blob/FileTransferJobs.\n"
  example: []
  syntax:
    content: 'public class TransferException : Exception'
    content.vb: >-
      Public Class TransferException
          Inherits Exception
  inheritance:
  - System.Object
  modifiers.csharp:
  - public
  - class
  modifiers.vb:
  - Public
  - Class
- uid: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor
  commentId: M:Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor
  id: '#ctor'
  parent: Microsoft.WindowsAzure.Storage.DataMovement.TransferException
  langs:
  - csharp
  - vb
  name: TransferException()
  fullName: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.TransferException()
  type: Constructor
  source:
    remote:
      path: lib/Exceptions/TransferException.cs
      branch: master
      repo: https://github.com/Azure/azure-storage-net-data-movement
    id: .ctor
    path: lib/Exceptions/TransferException.cs
    startLine: 47
  assemblies:
  - Microsoft.WindowsAzure.Storage.DataMovement
  packages:
  - Microsoft.WindowsAzure.Storage.DataMovement(0.3.0.0)
  namespace: Microsoft.WindowsAzure.Storage.DataMovement
  summary: "\nInitializes a new instance of the <xref href=\"Microsoft.WindowsAzure.Storage.DataMovement.TransferException\" data-throw-if-not-resolved=\"false\"></xref> class.\n"
  example: []
  syntax:
    content: public TransferException()
    content.vb: Public Sub New
  modifiers.csharp:
  - public
  modifiers.vb:
  - Public
- uid: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(System.String)
  commentId: M:Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: Microsoft.WindowsAzure.Storage.DataMovement.TransferException
  langs:
  - csharp
  - vb
  name: TransferException(String)
  fullName: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.TransferException(System.String)
  type: Constructor
  source:
    remote:
      path: lib/Exceptions/TransferException.cs
      branch: master
      repo: https://github.com/Azure/azure-storage-net-data-movement
    id: .ctor
    path: lib/Exceptions/TransferException.cs
    startLine: 55
  assemblies:
  - Microsoft.WindowsAzure.Storage.DataMovement
  packages:
  - Microsoft.WindowsAzure.Storage.DataMovement(0.3.0.0)
  namespace: Microsoft.WindowsAzure.Storage.DataMovement
  summary: "\nInitializes a new instance of the <xref href=\"Microsoft.WindowsAzure.Storage.DataMovement.TransferException\" data-throw-if-not-resolved=\"false\"></xref> class.\n"
  example: []
  syntax:
    content: public TransferException(string message)
    content.vb: Public Sub New(message As String)
    parameters:
    - id: message
      type: System.String
      description: The message that describes the error.
  modifiers.csharp:
  - public
  modifiers.vb:
  - Public
- uid: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(System.String,Exception)
  commentId: M:Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(System.String,Exception)
  id: '#ctor(System.String,Exception)'
  parent: Microsoft.WindowsAzure.Storage.DataMovement.TransferException
  langs:
  - csharp
  - vb
  name: TransferException(String, Exception)
  fullName: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.TransferException(System.String, Exception)
  type: Constructor
  source:
    remote:
      path: lib/Exceptions/TransferException.cs
      branch: master
      repo: https://github.com/Azure/azure-storage-net-data-movement
    id: .ctor
    path: lib/Exceptions/TransferException.cs
    startLine: 66
  assemblies:
  - Microsoft.WindowsAzure.Storage.DataMovement
  packages:
  - Microsoft.WindowsAzure.Storage.DataMovement(0.3.0.0)
  namespace: Microsoft.WindowsAzure.Storage.DataMovement
  summary: "\nInitializes a new instance of the <xref href=\"Microsoft.WindowsAzure.Storage.DataMovement.TransferException\" data-throw-if-not-resolved=\"false\"></xref> class.\n"
  example: []
  syntax:
    content: public TransferException(string message, Exception ex)
    content.vb: Public Sub New(message As String, ex As Exception)
    parameters:
    - id: message
      type: System.String
      description: The error message that explains the reason for the exception.
    - id: ex
      type: Exception
      description: >-
        The exception that is the cause of the current exception, or a null reference

        if no inner exception is specified.
  modifiers.csharp:
  - public
  modifiers.vb:
  - Public
- uid: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode)
  commentId: M:Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode)
  id: '#ctor(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode)'
  parent: Microsoft.WindowsAzure.Storage.DataMovement.TransferException
  langs:
  - csharp
  - vb
  name: TransferException(TransferErrorCode)
  fullName: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.TransferException(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode)
  type: Constructor
  source:
    remote:
      path: lib/Exceptions/TransferException.cs
      branch: master
      repo: https://github.com/Azure/azure-storage-net-data-movement
    id: .ctor
    path: lib/Exceptions/TransferException.cs
    startLine: 75
  assemblies:
  - Microsoft.WindowsAzure.Storage.DataMovement
  packages:
  - Microsoft.WindowsAzure.Storage.DataMovement(0.3.0.0)
  namespace: Microsoft.WindowsAzure.Storage.DataMovement
  summary: "\nInitializes a new instance of the <xref href=\"Microsoft.WindowsAzure.Storage.DataMovement.TransferException\" data-throw-if-not-resolved=\"false\"></xref> class.\n"
  example: []
  syntax:
    content: public TransferException(TransferErrorCode errorCode)
    content.vb: Public Sub New(errorCode As TransferErrorCode)
    parameters:
    - id: errorCode
      type: Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode
      description: Transfer error code.
  modifiers.csharp:
  - public
  modifiers.vb:
  - Public
- uid: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode,System.String)
  commentId: M:Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode,System.String)
  id: '#ctor(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode,System.String)'
  parent: Microsoft.WindowsAzure.Storage.DataMovement.TransferException
  langs:
  - csharp
  - vb
  name: TransferException(TransferErrorCode, String)
  fullName: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.TransferException(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode, System.String)
  type: Constructor
  source:
    remote:
      path: lib/Exceptions/TransferException.cs
      branch: master
      repo: https://github.com/Azure/azure-storage-net-data-movement
    id: .ctor
    path: lib/Exceptions/TransferException.cs
    startLine: 85
  assemblies:
  - Microsoft.WindowsAzure.Storage.DataMovement
  packages:
  - Microsoft.WindowsAzure.Storage.DataMovement(0.3.0.0)
  namespace: Microsoft.WindowsAzure.Storage.DataMovement
  summary: "\nInitializes a new instance of the <xref href=\"Microsoft.WindowsAzure.Storage.DataMovement.TransferException\" data-throw-if-not-resolved=\"false\"></xref> class.\n"
  example: []
  syntax:
    content: public TransferException(TransferErrorCode errorCode, string message)
    content.vb: Public Sub New(errorCode As TransferErrorCode, message As String)
    parameters:
    - id: errorCode
      type: Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode
      description: Transfer error code.
    - id: message
      type: System.String
      description: Exception message.
  modifiers.csharp:
  - public
  modifiers.vb:
  - Public
- uid: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode,System.String,Exception)
  commentId: M:Microsoft.WindowsAzure.Storage.DataMovement.TransferException.#ctor(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode,System.String,Exception)
  id: '#ctor(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode,System.String,Exception)'
  parent: Microsoft.WindowsAzure.Storage.DataMovement.TransferException
  langs:
  - csharp
  - vb
  name: TransferException(TransferErrorCode, String, Exception)
  fullName: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.TransferException(Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode, System.String, Exception)
  type: Constructor
  source:
    remote:
      path: lib/Exceptions/TransferException.cs
      branch: master
      repo: https://github.com/Azure/azure-storage-net-data-movement
    id: .ctor
    path: lib/Exceptions/TransferException.cs
    startLine: 99
  assemblies:
  - Microsoft.WindowsAzure.Storage.DataMovement
  packages:
  - Microsoft.WindowsAzure.Storage.DataMovement(0.3.0.0)
  namespace: Microsoft.WindowsAzure.Storage.DataMovement
  summary: "\nInitializes a new instance of the <xref href=\"Microsoft.WindowsAzure.Storage.DataMovement.TransferException\" data-throw-if-not-resolved=\"false\"></xref> class.\n"
  example: []
  syntax:
    content: public TransferException(TransferErrorCode errorCode, string message, Exception innerException)
    content.vb: Public Sub New(errorCode As TransferErrorCode, message As String, innerException As Exception)
    parameters:
    - id: errorCode
      type: Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode
      description: Transfer error code.
    - id: message
      type: System.String
      description: Exception message.
    - id: innerException
      type: Exception
      description: Inner exception.
  modifiers.csharp:
  - public
  modifiers.vb:
  - Public
- uid: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.ErrorCode
  commentId: P:Microsoft.WindowsAzure.Storage.DataMovement.TransferException.ErrorCode
  id: ErrorCode
  parent: Microsoft.WindowsAzure.Storage.DataMovement.TransferException
  langs:
  - csharp
  - vb
  name: ErrorCode
  fullName: Microsoft.WindowsAzure.Storage.DataMovement.TransferException.ErrorCode
  type: Property
  source:
    remote:
      path: lib/Exceptions/TransferException.cs
      branch: master
      repo: https://github.com/Azure/azure-storage-net-data-movement
    id: ErrorCode
    path: lib/Exceptions/TransferException.cs
    startLine: 132
  assemblies:
  - Microsoft.WindowsAzure.Storage.DataMovement
  packages:
  - Microsoft.WindowsAzure.Storage.DataMovement(0.3.0.0)
  namespace: Microsoft.WindowsAzure.Storage.DataMovement
  summary: "\nGets the detailed error code.\n"
  example: []
  syntax:
    content: public TransferErrorCode ErrorCode { get; }
    content.vb: Public ReadOnly Property ErrorCode As TransferErrorCode
    parameters: []
    return:
      type: Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode
      description: The error code of the exception.
  modifiers.csharp:
  - public
  - get
  modifiers.vb:
  - Public
  - ReadOnly
references:
- uid: Microsoft.WindowsAzure.Storage.DataMovement
  isExternal: false
  name: Microsoft.WindowsAzure.Storage.DataMovement
  fullName: Microsoft.WindowsAzure.Storage.DataMovement
- uid: Microsoft.WindowsAzure.Storage.DataMovement.TransferException
  parent: Microsoft.WindowsAzure.Storage.DataMovement
  isExternal: false
  name: TransferException
  fullName: Microsoft.WindowsAzure.Storage.DataMovement.TransferException
- uid: System.String
  isExternal: true
  name: String
  fullName: System.String
- uid: Exception
  isExternal: true
  name: Exception
  fullName: Exception
- uid: Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode
  parent: Microsoft.WindowsAzure.Storage.DataMovement
  isExternal: false
  name: TransferErrorCode
  fullName: Microsoft.WindowsAzure.Storage.DataMovement.TransferErrorCode
